test/deprecated/datetimeMinusDate.chpl

```diff
@@ -0,0 +1,6 @@
+use DateTime;
+
+const dt = datetime.now(),
+      d  = date.today();
+
+const td = dt - d;
```

Comment:
I think it'd probably be good to ensure the intended behavior of the deprecated symbol is still maintained until it is removed, so would like to see `td` verified in some way (though I recognize it is difficult with these symbols, given their reliance on the current time)

Reply:
Thanks!

I updated two of the four tests to add some functionality verification. The other two are less straightforward to verify though.